### PR TITLE
Add cascades

### PR DESCRIPTION
**Pull request by GDbateaux**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -116,3 +116,16 @@ ALTER TABLE visitor_beer_reviews ADD FOREIGN KEY (visitor_id) REFERENCES visitor
 ALTER TABLE cities ADD FOREIGN KEY (country_id) REFERENCES countries (id);
 
 ALTER TABLE festivals ADD FOREIGN KEY (city_id) REFERENCES cities (id);
+
+
+ALTER TABLE relation_bar_festival ADD CONSTRAINT fk_festival_cascade FOREIGN KEY (festival_id) REFERENCES festivals(id) ON DELETE CASCADE;
+ALTER TABLE relation_bar_festival ADD CONSTRAINT fk_bar_cascade FOREIGN KEY (bar_id) REFERENCES bars(id) ON DELETE CASCADE;
+
+ALTER TABLE cities ADD CONSTRAINT fk_country_cascade FOREIGN KEY (country_id) REFERENCES countries(id) ON DELETE CASCADE;
+ALTER TABLE festivals ADD CONSTRAINT fk_city_cascade FOREIGN KEY (city_id) REFERENCES cities(id) ON DELETE CASCADE;
```
Comment (MadeInShineA) [inline schema.sql:125]: I think this should be a ON DELETE SET NULL

**Review by MadeInShineA — CHANGES_REQUESTED** [review]
You should add the ON DELETE CASCADE for the other reviews as well

**Review by MadeInShineA — APPROVED** [review]
Perfect, thanks

**Review by NathanAnto — APPROVED** [review]
Tout bon

**Pull request merged**